ocaml/idl/datamodel_host.ml

```diff
@@ -2135,8 +2135,10 @@ let t =
         ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:Bool
             ~default_value:(Some (VBool false)) "https_only"
             "Reflects whether port 80 is open (false) or not (true)"
-        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:(Set update_guidances)
-            "recommended_guidances" ~default_value:(Some (VSet []))
+        ; field ~qualifier:DynamicRO ~internal_only:true
+            ~lifecycle:[(Prototyped, "23.18.0", ""); (Removed, "23.24.0", "")]
```

Comment:
The future is a strange place :)

Reply:
Yeah this never really works with backports.... removed in 23.19.1 and 23.24.0 really, but not in 23.20-23... What to do?

Reply:
you can say it got added back and removed again if you really want to document the intermediate state.